Documentation/install/upgrade.rst

```diff
@@ -112,14 +111,12 @@ Kubernetes resources are updated accordingly to version you are upgrading to:
 
 .. include:: ../gettingstarted/k8s-install-download-release.rst
 
-Generate the required YAML file and deploy it:
+Deploy helm release:
 
-.. code:: bash
+.. parsed-literal::
 
-   helm template cilium \
-     --namespace kube-system \
-     > cilium.yaml
-   kubectl apply -f cilium.yaml
+   helm install cilium/cilium --version |CHART_RELEASE| \\
```

Comment:
I don't think this will work for upgrade:

```
# helm install cilium cilium/cilium --version v1.6.5 --namespace=kube-system 
Error: rendered manifests contain a resource that already exists. Unable to continue with install: existing resource conflict: kind: ConfigMap, namespace: kube-system, name: cilium-config
```

Reply:
Replaced with `helm upgrade`